package org.challenge.exception;

public class SaveRecordException extends RuntimeException{
    public SaveRecordException(String message) {
        super(message);
    }
}
